motion_controller_pid:
  robot_frame: "robot_base_link"
  world_frame: 'world'
  control_loop_vel:
    vel_loop_freq: 50.0
    control_loop_vel_lin_hor:
      flag_ctr_vel_lin_hor: True
      PID:
        gains:
          gain_P: 1.0
          gain_I: 0.0
          gain_D: 0.0
        anti_wind_up: [-0.1, 0.1]
        saturation: [-1.0, 1.0]
    control_loop_vel_lin_z:
      flag_ctr_vel_lin_z: True
      PID:
        gains:
          gain_P: 1.0
          gain_I: 0.0
          gain_D: 0.0
        anti_wind_up: [-0.1, 0.1]
        saturation: [-1.0, 1.0]
    control_loop_vel_ang_z:
      flag_ctr_vel_ang_z: True
      PID:
        gains:
          gain_P: 1.0
          gain_I: 0.0
          gain_D: 0.0
        anti_wind_up: [-0.1, 0.1]
        saturation: [-1.0, 1.0]
  control_loop_pos:
    pos_loop_freq: 10.0
    control_loop_pos_hor:
      flag_ctr_pos_hor: True
      PID:
        gains:
          gain_P: 1.0
          gain_I: 0.0
          gain_D: 0.0
        anti_wind_up: [-0.1, 0.1]
        saturation: [-5.0, 5.0]
    control_loop_pos_ver:
      flag_ctr_pos_z: True
      PID:
        gains:
          gain_P: 1.0
          gain_I: 0.0
          gain_D: 0.0
        anti_wind_up: [-0.1, 0.1]
        saturation: [-5.0, 5.0]
    control_loop_pos_yaw:
      flag_ctr_att_yaw: True
      PID:
        gains:
          gain_P: 1.0
          gain_I: 0.0
          gain_D: 0.0
        anti_wind_up: [-0.1, 0.1]
        saturation: [-5.0, 5.0]


